src/tagmanager/tm_parser.h

```diff
@@ -1,6 +1,6 @@
 /*
 *
-*   Copyright (c) 2014, Colomban Wendling
+*   Copyright (c) 2005 The Geany contributors
```

Comment:
Here as well, although I'm surprised the content of this file is actually new since 2014, I'd probably have been moved here then instead, so maybe 2005 is fine indeed.

Reply:
Good catch, this was actually wrong as the file was first introduced in 2014 in https://github.com/geany/geany/commit/6a85a50d21b6297fe1ea51ecc8792f6a1f81cf58#diff-55056236371f88550f26cb87580ec0a4. Sorry, fixed.